tests/unit/modules/test_pkgng.py

```diff
@@ -539,3 +531,55 @@ def test_install_with_orphan(self):
                     python_shell=False,
                     env={},
                 )
+
+    def test_check_depends(self):
+        """
+        Test pkgng.check to check and install missing dependencies
+        """
+        pkg_cmd = MagicMock(return_value="")
+
+        with patch.dict(pkgng.__salt__, {"cmd.run": pkg_cmd}):
+            result = pkgng.check(depends=True)
+            self.assertEqual(result, "")
+            pkg_cmd.assert_called_with(
+                ["pkg", "check", "-dy"], output_loglevel="trace", python_shell=False,
+            )
+
+    def test_check_checksum(self):
+        """
+        Test pkgng.check for packages with invalid checksums
+        """
+        pkg_cmd = MagicMock(return_value="")
+
+        with patch.dict(pkgng.__salt__, {"cmd.run": pkg_cmd}):
+            result = pkgng.check(checksum=True)
+            self.assertEqual(result, "")
+            pkg_cmd.assert_called_with(
+                ["pkg", "check", "-s"], output_loglevel="trace", python_shell=False,
+            )
+
+    def test_check_recompute(self):
+        """
+        Test pkgng.check to recalculate the checksums of installed packages
+        """
+        pkg_cmd = MagicMock(return_value="")
+
+        with patch.dict(pkgng.__salt__, {"cmd.run": pkg_cmd}):
+            result = pkgng.check(recompute=True)
+            self.assertEqual(result, "")
+            pkg_cmd.assert_called_with(
+                ["pkg", "check", "-r"], output_loglevel="trace", python_shell=False,
+            )
+
+    def test_check_libs(self):
```

Comment:
```suggestion
    def test_checklibs(self):
```
To better associate with the parameter name?

Reply:
Native parameter is named `--shlibs`, but it doesn't sound too self-explanatory imho. For test case I put two underscores to separate method name and testcase, like `test_{method}_{flag}`, maybe `test_check_checklibs` makes more sense?

Reply:
in that case, yes, `test_check_checklibs` makes more sense.

Reply:
Fixed